### ISSUE TITLE
Generate default SSL certificate at runtime at first startup

### ISSUE THREAD

**Issue by thefosk** [Mashape/kong#453]

**Comment by thibaultCha** [comment]
+1

**Comment by thefosk** [comment]
Closed by mistake.

**Comment by thibaultCha** [comment]
Not related to the CLI this should be something specific to the SSL plugin, running in the init worker of the said plugin for example.

**Comment by thefosk** [comment]
This is a minor improvement, not a new feature, of the current SSL implementation, so it needs to happen where currently the weak point is, in the CLI.

As for changing it in the future, there currently is a problem. The plugins should be able to be installed without restarting/reloading the entire Kong cluster to make them effective. Embedding this functionality into the SSL plugin would change this behavior introducing an exception and requiring the cluster to be restarted/reloaded.

The reason why the SSL plugin currently works without restarting/reloading the cluster, is because SSL is enabled by default and what the SSL plugin does is only changing the SSL certificate being served.

In order to enable SSL by default, nginx requires default certificates, that will be auto-generated after this PR is being merged (as opposed as being hard coded). 

Basically enabling SSL is in the nginx scope, not in the plugin scope.
